src/solvespace.cpp

```diff
@@ -560,12 +560,18 @@ bool SolveSpaceUI::GetFilenameAndSave(bool saveAs) {
 
     if(saveAs || saveFile.IsEmpty()) {
         Platform::FileDialogRef dialog = Platform::CreateSaveFileDialog(GW.window);
+        // FIXME(emscripten):
+        dbp("Calling AddFilter()...");
```

Comment:
Should these `dbp` calls stay?

Reply:
@ruevs Oops, I forgot removing many debugging messages I've added; including it.